pages/about/monthly-calls.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Past monthly calls
+permalink: /about/monthly-calls/
+redirect_from:
+- /documentation/video-tutorials/
+- /about/monthly-call/
+layout: styleguide
+category: About
+lead: At USWDS monthly calls, we share important updates and tips about how to use the Design System.
+subnav:
+- text: Latest updates
+  href: '#changelog'
```

Comment:
I know it's a bit redundant given how small the page is but should we include the USWDS Playlist heading to the nav so it matches the in-page navigation?

```suggestion
subnav:
- text: USWDS playlist
  href: '#uswds-playlist'
- text: Latest updates
  href: '#changelog'
```

Reply:
Sounds good to me!